src/components/developers/WebhookPortal.tsx

```
@@ -0,0 +1,443 @@
+                  Subscribe to System Events *
+                </label>
+                <div className="space-y-2">
+                  {AVAILABLE_WEBHOOK_EVENTS.map((evt) => {
+                    const isChecked = selectedEvents.includes(evt.type);
+                    return (
+                      <div
+                        key={evt.type}
+                        onClick={() => handleToggleEvent(evt.type)}
+                        className={cn(
+                          "p-2.5 border-2 rounded-md cursor-pointer transition-all flex items-center justify-between text-xs",
+                          isChecked ? "border-black bg-sky-50 font-bold" : "border-gray-300 opacity-60"
```

ESLint

[error] Replace `·?·"border-black·bg-sky-50·font-bold"·:·"border-gray-300·opacity-60"` with `⏎····························?·"border-black·bg-sky-50·font-bold"⏎····························:·"border-gray-300·opacity-60",`

(prettier/prettier)